src/content/insertable-streams/video-processing/js/webgpu-background-blur.js

```diff
@@ -663,10 +685,65 @@ const batch = [4, 4];
 
     await device.queue.onSubmittedWorkDone();
 
+    const widthBit = Math.ceil(this.presentationWidth * 4 / 256)
+    const bytesPerRow = widthBit * 256;
+    const downloadBuffer = device.createBuffer({size: bytesPerRow * this.presentationHeight,
```

Comment:
Can `downloadBuffer` be reused instead of being created in every frame?

Reply:
Sure! Done.